crypto/curve25519/curve25519.c

```diff
@@ -909,65 +849,65 @@ static void fe_pow22523(fe out, const fe z) {
   fe t2;
   int i;
 
-  fe_sq(t0, z);
-  fe_sq(t1, t0);
+  GFp_fe_sq(t0, z);
```

Comment:
Since the time you wrote this PR, I changed the strategy for exporting the `fe_*` functions from C to Rust. Instead of renaming the original functions and making them `extern` (non-`static`), I have created an extern `GFp_fe_*()` wrapper around each static `fe_*()` function. Thus, the Rust code goes through the wrapper while the C code doesn't. Thus, we don't need to change any `fe_sq`/`fe_mul` callers in the C code any more

(I imagine that shortly after this PR, we will replace this function with Rust code anyway.)